apps/storybook/src/forms/ButtonGroup.stories.tsx

```diff
@@ -1,33 +1,112 @@
 import type { Meta, StoryObj } from "@storybook/react";
+import type { ComponentPropsWithRef } from "react";
 
-import { Button, ButtonGroup } from "@optiaxiom/react";
+import { Button, ButtonGroup, Flex } from "@optiaxiom/react";
 import { IconArrowRight, IconDownload, IconPhoto } from "@tabler/icons-react";
 
-const meta: Meta<typeof ButtonGroup> = {};
+const meta: Meta<typeof ButtonGroup> = {
+  component: ButtonGroup,
+};
 
 export default meta;
 
 type Story = StoryObj<typeof ButtonGroup>;
 
+const Content = ({
+  preset,
+}: Pick<ComponentPropsWithRef<typeof Button>, "preset">) => (
+  <>
+    <Button leftSection={<IconPhoto size={16} />} preset={preset}>
+      Gallery
+    </Button>
+    <Button preset={preset} rightSection={<IconDownload size={16} />}>
+      Download
+    </Button>
+    <Button
+      leftSection={<IconPhoto size={16} />}
+      preset={preset}
+      rightSection={<IconArrowRight size={16} />}
+    >
+      Enter Gallery
+    </Button>
+  </>
+);
+
+const BasicContent = ({
+  children,
+  ...props
+}: Pick<
+  ComponentPropsWithRef<typeof Button>,
+  "children" | "colorScheme" | "preset" | "variant"
+>) => (
+  <>
+    <Button {...props}>{children} One</Button>
+    <Button {...props}>{children} Two</Button>
+  </>
+);
+
 export const Horizontal: Story = {
-  render: () => (
-    <ButtonGroup>
-      <Button leftSection={<IconPhoto />}>Gallery</Button>
-      <Button rightSection={<IconDownload />}>Download</Button>
-      <Button leftSection={<IconPhoto />} rightSection={<IconArrowRight />}>
-        Enter Gallery
-      </Button>
-    </ButtonGroup>
-  ),
+  args: {
+    children: <Content />,
+  },
 };
+
 export const Vertical: Story = {
-  render: () => (
-    <ButtonGroup orientation="vertical">
-      <Button leftSection={<IconPhoto />}>Gallery</Button>
-      <Button rightSection={<IconDownload />}>Download</Button>
-      <Button leftSection={<IconPhoto />} rightSection={<IconArrowRight />}>
-        Enter Gallery
-      </Button>
-    </ButtonGroup>
+  args: {
+    children: <Content />,
+    orientation: "vertical",
+  },
+};
+
+export const Spacing: Story = {
+  args: {
+    children: <Content />,
+    gap: "8",
+  },
+};
+
+const presets = [
+  ["default", "Default"],
+  ["primary", "Primary"],
+  ["danger", "Danger"],
+  ["danger-outline", "Outline"],
+  ["secondary", "Secondary"],
+] as const;
+
+export const Presets: Story = {
```

Comment:
Should ghost buttons have border between them?

<img width="403" alt="image" src="https://github.com/optimizely-axiom/optiaxiom/assets/8050659/3218ee71-d778-4eef-8247-9b43446c293e">

Reply:
still debating that with designers